docs/tempo/website/getting-started/_index.md

```diff
@@ -5,9 +5,79 @@ weight: 100
 
 # Getting started with Tempo
 
+Distributed tracing is used to visualise the lifecycle of a request as it passes through
+an application. There are a few components that need to be set up in order to get a
+working distributed tracing visualisation.
+
+## 1. Client instrumentation
```

Comment:
As Joe mentioned at the meeting, probably we don't want to make special emphasis on OpenTelemetry.

Maybe we can have the following structure in this section:

```
## 1. Client instrumentation
Brief intro to instrumentation
### OpenTelemetry
Describe and point to OTEL stuff. Probably linking to all the SDKs is a bit overkill. We can mention here those two approaches that you can follow with OTEL: manual or auto.
### Jaeger/OpenTracing/Zipkin/Whatever
Small text
### Jaeger/OpenTracing/Zipkin/Whatever
Small text
### Others
```

Just having a small paragraph and some pointers around the most used ones.